lib/cli.go

```diff
@@ -25,6 +25,7 @@ import (
 
 	"github.com/purpleidea/mgmt/bindata"
 	"github.com/purpleidea/mgmt/gapi"
+
```

Comment:
nack this nl change... this might be due to change in gofmt, we're still on 1.10.x but will be moving to 1.11.x shortly. For now, please don't include this.

Reply:
amended the commit